src/Microsoft.ML.Core/Data/MetadataUtils.cs

```diff
@@ -175,7 +175,7 @@ internal static KeyType ScoreColumnSetIdType
             {
                 if (_scoreColumnSetIdType == null)
                 {
-                    var type = new KeyType(DataKind.U4, 0, 0);
+                    var type = new KeyType(DataKind.U4, 0);
```

Comment:
>KeyType(DataKind.U4, 0); [](start = 35, length = 24)

Interesting. I think we may have to do something else here. Maybe the count becomes, say, `int.MaxValue - 1` or something.